rag/llm/chat_model.py

```diff
@@ -158,6 +158,28 @@ def chat(self, system, history, gen_conf):
             return "**ERROR**: " + str(e), 0
 
 
+class XinferenceChat(Base):
+    def __init__(self, key=None, model_name="", base_url=""):
```

Comment:
If it was me, i would use `model_uid` instead of `model_name` here because `xinference` requires `model_uid`. Reference: [here](https://inference.readthedocs.io/zh-cn/latest/user_guide/client_api.html#xinference-client). 

I saw frontend have added deal with it in [here](https://github.com/infiniflow/ragflow/pull/319/files#diff-aff607f35b04cd97cc52de27e84d0fad71c895582996a068adf08c6b99ebfede)